Scheduler/observing_blocks_auxtel/latiss_densePTC_calibrations.json

```diff
@@ -0,0 +1,553 @@
+{
+    "name": "latiss_daily_calibrations",
+    "program": "latiss_daily_calibrations",
+    "constraints": [],
+    "scripts": [
+        {
+            "name": "set_summary_state.py",
+            "standard": true,
+            "parameters": {
+                "data": [
+                    [
+                        "ATWhiteLight",
+                        "ENABLED"
+                    ]
+                ]
+            }
+        },
+        {
+            "name": "set_summary_state.py",
+            "standard": true,
+            "parameters": {
+                "data": [
+                    [
+                        "OCPS:1",
+                        "ENABLED"
+                    ]
+                ]
+            }
+        },
+        {
+            "name": "auxtel/calibrations/power_on_atcalsys.py",
+            "standard": true,
+            "parameters": {}
+        },
+        {
+            "name": "auxtel/prepare_for/flat.py",
+            "standard": true,
+            "parameters": {}
+        },
+        {
+            "name": "auxtel/make_latiss_calibrations.py",
+            "standard": false,
+            "parameters": {
+                "n_flat": 20,
+                "exp_times_flat": 6,
+                "script_mode": "BIAS_DARK_FLAT",
+                "filter": "SDSSr_65mm",
+                "grating": "empty_1"
+            }
+        },
+        {
+            "name": "auxtel/make_latiss_calibrations.py",
+            "standard": false,
+            "parameters": {
+                "n_bias": 3,
+                "n_dark": 3,
+                "exp_times_dark": 6,
+                "n_flat": 20,
+                "exp_times_flat": 6,
+                "script_mode": "BIAS_DARK_FLAT",
+                "filter": "SDSSg_65mm",
+                "grating": "empty_1"
+            }
+        },
+        {
+            "name": "auxtel/make_latiss_calibrations.py",
+            "standard": false,
+            "parameters": {
+                "n_bias": 3,
+                "n_dark": 3,
+                "exp_times_dark": 6,
+                "n_flat": 20,
+                "exp_times_flat": 3,
+                "script_mode": "BIAS_DARK_FLAT",
+                "filter": "SDSSz_65mm",
+                "grating": "empty_1"
```

Comment:
Since you are just looking for the dense PTC, you can skip these scripts which take flats in different filters.